android/library/src/main/java/io/logto/android/Logto.kt

```diff
@@ -41,71 +44,98 @@ class Logto(
     val authenticated: Boolean
         get() = tokenSet != null
 
-    fun getAccessToken(block: (accessToken: String) -> Unit) = tokenSet?.let {
-        if (!it.isExpired()) {
-            block(it.accessToken)
-            return@let
+    fun getAccessToken(
+        block: HandleAccessTokenCallback
+    ) {
+        if (tokenSet == null) {
+            block(LogtoException(LogtoException.NOT_AUTHENTICATED), null)
```

Comment:
since both `exception` & `accessToken`  attributes are optional, can we simply return the `exception` without filling in `null` for the `accessToken` attribute?

Reply:
kotlin does not support default value of parameter in function type